Remember The Date/View Controllers/ZenHelpViewController.m

```diff
@@ -95,6 +96,24 @@ -(void) setupSupportInformation {
     }];
 }
 
+- (void) setupSDK {
+    static dispatch_once_t onceToken;
```

Comment:
Is this a good idea?

If the init code only gets called once per app life cycle, but this controller can be inited multiple times so the `_sdkIsready` property is able to be `NO` even when init has been done.

A bit ranty, does it make sense?